Dockerfile

```
@@ -1,4 +1,23 @@
-FROM tgstation/byond:513.1533 as base
+FROM i386/ubuntu:xenial as base
+
+ARG BYOND_MAJOR=514
+ARG BYOND_MINOR=1589
+
+RUN apt-get update \
+    && apt-get install -y \
+    curl \
+    unzip \
+    make \
+    libstdc++6 \
+    && curl "http://www.byond.com/download/build/${BYOND_MAJOR}/${BYOND_MAJOR}.${BYOND_MINOR}_byond_linux.zip" -o byond.zip \
```

As HTTPS is available for downloading the linux build:

```suggestion
    && curl "https://www.byond.com/download/build/${BYOND_MAJOR}/${BYOND_MAJOR}.${BYOND_MINOR}_byond_linux.zip" -o byond.zip \
```

---

Good catch.

I haven't had any issue with finding the config files on linux, but there's been one error visible when launching the containers with the database connection. Seemingly not finding the libmariadb.so file:

```
2023/09/19 13:58:45 | stderr | Your server failed to establish a connection with the SQL database.
2023/09/19 13:58:45 | stderr | libmariadb.so: cannot open shared object file: No such file or directory
2023/09/19 13:58:45 | stderr | Your server failed to establish a connection with the feedback database.
2023/09/19 13:58:45 | stderr | libmariadb.so: cannot open shared object file: No such file or directory
```